webui/src/utils.ts

```diff
@@ -46,16 +46,18 @@ export function toLocalTime(timestamp?: string): Date | undefined {
     return utcToZonedTime(timestamp, timeZone);
 }
 
-export function handleError(err?: Error | Partial<ErrorResponse>): void {
+export function handleError(err?: Error | Partial<ErrorResponse>): string {
+    let errorMessage: string = '';
```

Comment:
In case `err` is undefined you should return something like "An unexpected error occurred."